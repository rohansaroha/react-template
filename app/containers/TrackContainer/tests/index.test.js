/**
 *
 * Tests for TrackContainer
 *
 *
 */

import React from 'react';
import { renderProvider } from '@utils/testUtils';
// import { fireEvent } from '@testing-library/dom'
import { TrackContainerTest as TrackContainer } from '../index';

describe('<TrackContainer /> container tests', () => {
  // let submitSpy

  beforeEach(() => {
    // submitSpy = jest.fn()
  });
  it('should render and match the snapshot', () => {
    const { baseElement } = renderProvider(<TrackContainer />);
    expect(baseElement).toMatchSnapshot();
  });
});
