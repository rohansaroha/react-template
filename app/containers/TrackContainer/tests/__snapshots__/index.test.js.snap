// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TrackContainer /> container tests should render and match the snapshot 1`] = `
<body>
  <div>
    <div
      class="ant-card SoundCard__CustomCard-lvsa1p-0 guwSkq ant-card-bordered"
      data-testid="sound-card"
    >
      <div
        class="ant-card-body"
      >
        <div
          data-testid="link-track"
        >
          <div
            class="SoundCard__HeaderBox-lvsa1p-2 elgeDp"
          >
            <img
              alt="'Till I Collapse (feat. Nate Dogg)"
              class="SoundCard__AudioImg-lvsa1p-4 jRZWqW"
              src="https://is5-ssl.mzstatic.com/image/thumb/Music115/v4/61/d5/0a/61d50a3d-4a27-187a-d16f-6b8ce4b62560/source/100x100bb.jpg"
            />
            <div
              style="flex: 4;"
            >
              <div
                class="SoundCard__SongPrimary-lvsa1p-3 djMiVS"
              >
                'Till I Collapse (feat. Nate Dogg)
              </div>
              <div
                class="SoundCard__SongSecondary-lvsa1p-5 bBBuVf"
              >
                Eminem
              </div>
              <button
                class="ant-btn SoundCard__ButtonSong-lvsa1p-7 gQYgvp ant-btn-dangerous"
                data-testid="play-button"
                type="button"
              >
                <span>
                  Play full Song
                </span>
              </button>
            </div>
          </div>
        </div>
        <audio
          class="SoundCard__AudioBox-lvsa1p-6 gwtEvh"
          controls=""
        >
          <source
            src="https://audio-ssl.itunes.apple.com/itunes-assets/AudioPreview123/v4/9e/cb/06/9ecb061b-d98c-575e-6649-cd9d472c1eb9/mzaf_18237082239512341748.plus.aac.p.m4a"
          />
        </audio>
      </div>
    </div>
  </div>
</body>
`;
